pkgs/tools/misc/kisslicer/default.nix

```diff
@@ -0,0 +1,55 @@
+{ fetchurl
+, libX11
+, mesa
+, makeWrapper
+, stdenv
+, unzip
+, inidir ? "\\\$HOME/.config/kisslicer"
```

Comment:
I'd suggest to either change the default here to `${XDG_CONFIG_HOME:-$HOME/.config}/kisslicer` or to always use this, i.e., not to have an `inidir` parameter.

Reply:
Done.